2023/01.md

```diff
@@ -14,7 +14,7 @@
 For meeting times in your timezone, visit [Temporal docs](https://tc39.es/proposal-temporal/docs/) and run the code below in the devtools console.
 
 ```js
-Temporal.ZonedDateTime.from('2023-01-20T10:00[America/New_York]')
+Temporal.ZonedDateTime.from('2023-01-30T10:00[America/New_York]')
```

Comment:
I'm confused, #1293 should have already set this to the 30th, what set it to the 20th?

Reply:
Yes. The commit was accidental. I could swear it's a bug in the GitHub UI...